libipl/libipl.H

```diff
@@ -59,7 +60,7 @@ const err_msg_map_type err_msg_map = {
     {IPL_ERR_SPARE_CLK, "Spare Clock initialization is failed"},
     {IPL_ERR_INVALID_NUM_CLOCK, "Invalid number of clock targets found"},
     {IPL_ERR_ATTR_WRITE, "Device tree attribute write failed"},
-};
+    {IPL_ERR_GUARD_FILE_READ, "Guard file partition corrupted"}};
```

Comment:
The error message is not correct, right? You are defining it for the read failure but, an error message saying it is corrupted.